### PR TITLE
Adding activities

### PR DESCRIPTION
**Pull request by ChristinaLK**
I'm adding to the instructor's notes, mainly: 
* motivation and some sample intro slides
* a way I've written queries up on the board
* an interactive activity I've done

### CODE REVIEW

File: instructor-notes.md [hunk 1]
```diff
@@ -27,6 +27,21 @@ The way to fix this is to:
 
 You can then use **Ctrl - +** to zoom just like any other web page.
 
+## Motivation and Framing
+
+See this slide deck as a sample intro for the lesson: 
+[SQL Intro Deck](https://speakerdeck.com/christinalk/data-carpentry-sql-introduction)
```
Comment (tpoisot) [inline instructor-notes.md]: I LOVE these slides.

**Comment by tpoisot** [comment]
I would be happy to merge -- @orchid00 ?

File: instructor-notes.md (hunk 1 above)
Comment (ChristinaLK) [inline instructor-notes.md]: thanks!  :)

**Comment by orchid00** [comment]
Thank you @ChristinaLK Very nice slides, 
It is good to merge.

**Pull request merged**